lib/logstash/outputs/bson/big_decimal.rb

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2009-2014 MongoDB Inc.
```

Comment:
I am leaving the comments in the files as is. See [Apache 2 license](http://www.apache.org/licenses/LICENSE-2.0) clause 4: Redistribution, paragraph c.